### PR TITLE
Feat/seedlot feedback page

### PR DESCRIPTION
**Pull request by ArthurEncr**
# Description

Seedlot created feedback page created.

[Jira ticket 
](https://apps.nrs.gov.bc.ca/int/jira/browse/FSADT2-477)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have already been accepted and merged

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Frontend](https://nr-spar-webapp-46-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar-webapp/actions/workflows/merge-main.yml)

### CODE REVIEW

**Review by RMCampos — APPROVED** [review]
LGTM

File: src/views/SeedlotCreatedFeedback/index.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          kind="tertiary"
+          onClick={() => null}
+          size="lg"
+          className="btn-landing"
+        >
+          Go back to seedlot’s main screen
+        </Button>
+      </Column>
+    </Row>
+
```
Comment (mgaseta) [inline src/views/SeedlotCreatedFeedback/index.tsx:49]: Remove empty line.

File: src/views/SeedlotCreatedFeedback/styles.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
+@use '../../theme/variables.scss' as vars;
```
Comment (mgaseta) [inline src/views/SeedlotCreatedFeedback/styles.scss:1]: For the whole page, add an space before the opening bracket on the selectors.

File: src/views/SeedlotCreatedFeedback/styles.scss [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+.scf-pic-container{
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.scf-pictogram{
+  width: 23.125rem;
+  height: 22.375rem;
+  color: var(--#{vars.$bcgov-prefix}-focus);
```
Comment (mgaseta) [inline src/views/SeedlotCreatedFeedback/styles.scss:20]: The color defined on the figma file is the primary button color (Light Theme/Button/$button-primary), adjust it here to reflect correctly the design.

File: src/views/SeedlotCreatedFeedback/styles.scss [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+.scf-info-container{
+  display: flex;
+  flex-direction: column;
+  padding-left: 7.563rem;
+}
```
Comment (mgaseta) [inline src/views/SeedlotCreatedFeedback/styles.scss:27]: Put an empty line in between rules.

File: src/views/SeedlotCreatedFeedback/styles.scss [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+.seedlot-created-feedback-page h2{
+  @include type.type-style('fluid-paragraph-01', true);
+  color: var(--#{vars.$bcgov-prefix}-text-secondary)
+}
+
+@media only screen and (max-width: 1584px) {
+.scf-pic-container{
+  justify-content: center;
+}
```
Comment (mgaseta) [inline src/views/SeedlotCreatedFeedback/styles.scss:45]: Adjust tabs here.

File: src/views/SeedlotCreatedFeedback/styles.scss [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+.seedlot-created-feedback-page{
+  margin: 0px;
+  height: auto;
+}
+
+.scf-info-container{
+  padding-left: 0px;
+  align-items: center;
+  padding: 3rem;
+}
```
Comment (mgaseta) [inline src/views/SeedlotCreatedFeedback/styles.scss:58]: Adjust tabs here.

File: src/views/SeedlotDashboard/styles.css [hunk 7]
```diff
@@ -1,5 +1,7 @@
 .seedlot-dashboard-content {
   padding-left: 2.5rem;
+  color: var(--bcgov-focus);
```
Comment (mgaseta) [inline src/views/SeedlotDashboard/styles.css:3]: Why change the color here?

**Review by mgaseta — CHANGES_REQUESTED** [review]
Good job 🚀✔! Just added a few markups, mostly related to code patterns, so no worries.

File: src/App.tsx [hunk 8]
```diff
@@ -48,6 +49,15 @@ const App: React.FC = () => {
               </ProtectedRoute>
             )}
           />
+
+          <Route
+            path="/seedlot-feedback"
```
[the remark this answers is not in the document]
Comment (RMCampos) [reply src/App.tsx:54]: Great point! I would suggest `/seedlot/successfully-created`.

**Review by lucasbuenoencora — CHANGES_REQUESTED** [review]
LGTM! I guess that after the merge the new component is on the wrong directory, just need to put it inside the frontend folder.

**Review by lucasbuenoencora — APPROVED** [review]
Great Job :rocket: :heavy_check_mark:

**Pull request merged**